.github/workflows/build_windows_packages.yml

```diff
@@ -114,6 +114,15 @@ jobs:
           git config --global core.longpaths true
           python ./build_tools/fetch_sources.py --jobs 96
 
+      - name: Verify hip-tests submodule (cmd)
+        shell: cmd
+        run: |
+          if not exist core\hip-tests\samples\1_Utils\hipInfo\hipInfo.cpp (
+            echo hip-tests submodule missing!
+            git submodule status
+            exit /b 1
+          )
+
```

Comment:
Form your testing: You need to patch fetch_sources. Just modifying gitmodules isn't sufficient. See https://github.com/ROCm/TheRock/blob/main/docs/development/git_chores.md#adding-a-new-submodule.